src/applications/vaos/tests/appointment-list/components/PastAppointmentsListV2.unit.spec.jsx

```diff
@@ -331,7 +344,7 @@ describe('VAOS <PastAppointmentsListV2>', () => {
     });
 
     await screen.findAllByText(
-      new RegExp(startDate.tz('America/Denver').format('dddd, MMMM D'), 'i'),
+      new RegExp(startDate.format('dddd, MMMM D'), 'i'),
```

Comment:
Express Care requests have apparently never adjusted their created dates by the facility zone, since we weren't ever showing dates for them before. So this tz adjustment was breaking things.